docs/examples/15_minutes_to_QCoDeS.ipynb

```diff
@@ -55,7 +55,19 @@
    "cell_type": "code",
    "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "ename": "ModuleNotFoundError",
+     "evalue": "No module named 'qcodes'",
+     "output_type": "error",
```

Comment:
Looks like something strange happened when you executed the notebook. Can you reexecute?

Reply:
I also realized something had gone wrong when PR showed few hundred lines are deleted.